todo_app/data/api.py

```diff
@@ -0,0 +1,114 @@
+from configparser import ConfigParser
+from todo_app.flask_config import Config
+from .boards import *
+import os
+import requests
+
+
+class TrelloAPI():
+    
+    def __init__(self):
+        self.TRELLO_URL = 'https://api.trello.com/1'
+        self.key_and_token = {'key':Config().TRELLO_KEY,'token':Config().TRELLO_TOKEN}
+        self.boardId = Config().BOARD_ID
+        self.to_do_id = Config().TO_DO_ID
+        self.doing_id = Config().DOING_ID
+        self.done_id = Config().DONE_ID
+        print("API board id:")
```

Comment:
I'm guessing these were to help with debugging - we should clear those out now

Reply:
Done